android/tests/espresso/com/frostwire/android/test/utils/PreferencesManipulator.java

```diff
@@ -0,0 +1,42 @@
+package com.frostwire.android.test.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import com.frostwire.android.core.Constants;
+
+import java.util.Map;
+
+public class PreferencesManipulator {
+    SharedPreferences mPreferences;
+
+    public PreferencesManipulator(Context context) {
+        mPreferences = PreferenceManager.getDefaultSharedPreferences(context);
+    }
+
+    public void clearGuiPrefs() {
+        clearPrefsWithPrefix("frostwire.prefs.gui.");
+    }
+
+    public void userHasStartedAppOnce() {
+        mPreferences.edit()
```

Comment:
the name of this method is a bit confusing.

It almost sounds as a question (even though it doesn't return a boolean), I suggest to rename it to something like:

`public void markAppAsUsed()`

IntelliJ renaming functionality will change the name here and every where else for you.